docs/release-notes/self-hosted/self-hosted-v8.2.0.md

```diff
@@ -0,0 +1,139 @@
+---
+rss_title: Codacy release notes RSS feed
+rss_href: /feed_rss_created.xml
+description: Release notes for Codacy Self-hosted v8.2.0.
+codacy_tools_version_old: https://github.com/codacy/codacy-tools/releases/tag/6.1.25
+codacy_tools_version_new: https://github.com/codacy/codacy-tools/releases/tag/6.3.0
+---
+
+# Self-hosted v8.2.0
+
+These release notes are for [Codacy Self-hosted v8.2.0](https://github.com/codacy/chart/releases/tag/8.2.0){: target="_blank"}, released on September 21, 2022.<!-- TODO Update release date -->
+
+📢 [Visit the Codacy roadmap](https://roadmap.codacy.com) and <span class="skip-vale">let us know</span> your feedback on both new and planned product updates!
+
+## Upgrading Codacy Self-hosted
+
+Follow the steps below to upgrade to Codacy Self-hosted v8.2.0:
+
+1.  Check the [release notes for all Codacy Self-hosted versions](../index.md#self-hosted) **between your current version and the most recent version** for breaking changes and follow the instructions provided <span class="skip-vale">carefully</span>.
+
+1.  Follow the instructions to [upgrade your Codacy Self-hosted instance](https://docs.codacy.com/v8.2/chart/maintenance/upgrade/).
+
+1.  Update your Codacy command-line tools to the versions with the Git tag `self-hosted-8.2.0`:
+
+    -   [Codacy Analysis CLI](https://github.com/codacy/codacy-analysis-cli/releases/tag/self-hosted-8.2.0)
+    -   [Codacy Coverage Reporter](https://github.com/codacy/codacy-coverage-reporter/releases/tag/self-hosted-8.2.0)
+
+## Breaking changes
+
+This version of Codacy Self-hosted requires [PostgreSQL version 10.20](https://docs.codacy.com/v8.2/chart/requirements/#postgresql-server-setup) since the previous major version [will reach end-of-life (EOL) later this year](https://www.postgresql.org/support/versioning/).
```

Comment:
Since we have this breaking change, the new Self-hosted release should increment the MAJOR version (9.0.0 instead of 8.2.0). This is [being discussed on Slack](https://codacy.slack.com/archives/CSSSHAK9N/p1663773818817719?thread_ts=1663751322.000309&cid=CSSSHAK9N).